### PR TITLE
Paulina/2 Data persistence

### PR DESCRIPTION
**Pull request by PaulinaLL**

### CODE REVIEW

File: routes/students.js [hunk 1]
```diff
@@ -1,60 +1,71 @@
...
-    name: "Rupert",
-    lastname: "Jalili",
-    age: 30,
-    class: "FBW101",
-    location: "BER"
-  }
-];
+const fs = require("fs");
+const path = require("path");
+const myPath = path.join(__dirname, "../data/students.json");
```
Comment (oliverwebr) [inline routes/students.js]: studentsData or something like this would be a nicer name. Not that the kids fight about who's path it is :P

Comment (PaulinaLL) [reply routes/students.js]: hehee studentsData insted of myPath done!

Comment (PaulinaLL) [reply routes/students.js]: i had a look on the conflict but i'm not sure what can i adjust to not break this beauty! :sweat_smile:

**Comment by oliverwebr** [comment]
fixes #3

**Pull request merged**